src/libraries/System.Console/src/System/Console.cs

```diff
@@ -374,27 +386,32 @@ public static void SetWindowSize(int width, int height)
 
         public static int LargestWindowWidth
         {
+            [UnsupportedOSPlatform("browser")]
             get { return ConsolePal.LargestWindowWidth; }
         }
 
         public static int LargestWindowHeight
         {
+            [UnsupportedOSPlatform("browser")]
             get { return ConsolePal.LargestWindowHeight; }
         }
 
+        [UnsupportedOSPlatform("browser")]
         public static bool CursorVisible
         {
             [SupportedOSPlatform("windows")]
             get { return ConsolePal.CursorVisible; }
             set { ConsolePal.CursorVisible = value; }
         }
```

Comment:
This is an interesting scenario!  @buyaa-n can you weigh in on how this will be interpreted with the property being marked as `[Unsupported("browser")]` but the `get` method having `[SupportedOSPlatform("windows")]`?  Is this OK or should we instead mark the `set` method as `[UnsupportedOSPlatform("browser")]`?

Reply:
Is the `UnsupportedOSPlatform` even necessary? I imagine that the behavior of the analyzer would be "if there is a `SupportedOSPlatformAttribute`" it will flag the API as unsupported on any other platforms? 

Maybe we should just mark the whole property as `SupportedOSPlatform("windows")`? It seems weird to be able to set a property that you can't get on the same platform? Or does this property affects other APIs that are supported?

Reply:
What was intended here was to mark the property getter as supported only on Windows, but the property _setter_ as supported everywhere (except browser now).  The getter and setter can have their own attributes.  But I don't know what the analyzer will do when the getter/setter differ from the property itself like this.

It might be clearer to just mark the `set` method as `[UnsupportedOSPlatform("browser")]` even if this would actually work as-is.

Reply:
I see. Maybe something to consider on the Analyzer? That if the property has an attribute, but the getter/setter has another, the getter/setter wins?

Reply:
> This is an interesting scenario! @buyaa-n can you weigh in on how this will be interpreted with the property being marked as [Unsupported("browser")] but the get method having [SupportedOSPlatform("windows")]? Is this OK or should we instead mark the set method as [UnsupportedOSPlatform("browser")]?

Yes it would be considered as an inconsistent list, as you mentioned the [UnsupportedOSPlatform("browser")] attribute better to be in the setter.

Reply:
> 

> I see. Maybe something to consider on the Analyzer? That if the property has an attribute, but the getter/setter has another, the getter/setter wins?

It works that way in case they are for the same platform, but not for different platform

Reply:
Cool. @mdh1418 per these comments, please update all properties where there was already `[SupportedOSPlatform("windows")]` on one of the accessors but not the other, and move the `[UnsupportedOSPlatform("browser")]` attribute down to the other accessor instead of on the property itself.